.env.example

```diff
@@ -29,6 +29,8 @@ CSAF_APP_EXTERNAL_PORT=4180
 CSAF_KEYCLOAK_EXTERNAL_PROTOCOL=http
```

Comment:
I think we can remove the variables  CSAF_KEYCLOAK_EXTERNAL_PROTOCOL, CSAF_KEYCLOAK_EXTERNAL_HOSTNAME and CSAF_KEYCLOAK_EXTERNAL_PORT as they are no longer needed

Reply:
Yes, I removed the variables.